src/DssSpell.t.sol

```diff
@@ -61,45 +37,27 @@ contract DssSpellTest is DssSpellTestBase {
-        uint256 prevSin = vat.sin(address(vow));
-
-        // Recognized Delegates
-        uint256 prevDaiFlipFlopFlap  = dai.balanceOf(FLIPFLOPFLAP);
-        uint256 prevDaiUltraSchuppi  = dai.balanceOf(ULTRASCHUPPI);
-        uint256 prevDaiFeedBlack     = dai.balanceOf(FEEDBLACKLOOPS);
-        uint256 prevDaiMakerMan      = dai.balanceOf(MAKERMAN);
-        uint256 prevDaiAcreInvest    = dai.balanceOf(ACREINVEST);
-        uint256 prevDaiMonetSupply   = dai.balanceOf(MONETSUPPLY);
-        uint256 prevDaiJustinCase    = dai.balanceOf(JUSTINCASE);
-        uint256 prevDaiGfxLabs       = dai.balanceOf(GFXLABS);
-        uint256 prevDaiDoo           = dai.balanceOf(DOO);
-        uint256 prevDaiFlipside      = dai.balanceOf(FLIPSIDE);
-
-        uint256 amount = amountFlipFlopFlap + amountUltraSchuppi + amountFeedBlack
-        + amountMakerMan + amountAcreInvest + amountMonetSupply  + amountJustinCase
-        + amountGfxLabs + amountDoo + amountFlipside;
+    function testPayments() private { // make public to use
+        // uint256 prevSin = vat.sin(address(vow));
 
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+        // // Recognized Delegates
+        // uint256 prevDaiFlipFlopFlap  = dai.balanceOf(FLIPFLOPFLAP);
+        // uint256 prevDaiUltraSchuppi  = dai.balanceOf(ULTRASCHUPPI);
+
+        // uint256 amount = amountFlipFlopFlap + amountUltraSchuppi;
 
-        assertEq(vat.sin(address(vow)) - prevSin, amount * RAD);
-
-        // Recognized Delegates
-        assertEq(dai.balanceOf(FLIPFLOPFLAP)   - prevDaiFlipFlopFlap, amountFlipFlopFlap * WAD);
-        assertEq(dai.balanceOf(FEEDBLACKLOOPS) - prevDaiFeedBlack,    amountFeedBlack    * WAD);
-        assertEq(dai.balanceOf(ULTRASCHUPPI)   - prevDaiUltraSchuppi, amountUltraSchuppi * WAD);
-        assertEq(dai.balanceOf(MAKERMAN)       - prevDaiMakerMan,     amountMakerMan     * WAD);
-        assertEq(dai.balanceOf(ACREINVEST)     - prevDaiAcreInvest,   amountAcreInvest   * WAD);
-        assertEq(dai.balanceOf(MONETSUPPLY)    - prevDaiMonetSupply,  amountMonetSupply  * WAD);
-        assertEq(dai.balanceOf(JUSTINCASE)     - prevDaiJustinCase,   amountJustinCase   * WAD);
-        assertEq(dai.balanceOf(GFXLABS)        - prevDaiGfxLabs,      amountGfxLabs      * WAD);
-        assertEq(dai.balanceOf(DOO)            - prevDaiDoo,          amountDoo          * WAD);
-        assertEq(dai.balanceOf(FLIPSIDE)       - prevDaiFlipside,     amountFlipside     * WAD);
+        // vote(address(spell));
+        // scheduleWaitAndCast(address(spell));
+        // assertTrue(spell.done());
+
+        // assertEq(vat.sin(address(vow)) - prevSin, amount * RAD);
+
+        // // Recognized Delegates
+        // assertEq(dai.balanceOf(FLIPFLOPFLAP)   - prevDaiFlipFlopFlap, amountFlipFlopFlap * WAD);
+        // assertEq(dai.balanceOf(FEEDBLACKLOOPS) - prevDaiFeedBlack,    amountFeedBlack    * WAD);
```

Comment:
I would make this general and comment it out or just remove the substance:

```suggestion
        // uint256 prevDaiWALLET0  = dai.balanceOf(<WALLET0>);
        // uint256 prevDaiWALLET1  = dai.balanceOf(<WALLET1>);

        // uint256 amount = prevDaiWALLET0 + prevDaiWALLET1;

        // vote(address(spell));
        // scheduleWaitAndCast(address(spell));
        // assertTrue(spell.done());

        // assertEq(vat.sin(address(vow)) - prevSin, amount * RAD);

        // // Recognized Delegates
        // assertEq(dai.balanceOf(WALLET0) - prevDaiWALLET0, amountWALLET0 * WAD);
        // assertEq(dai.balanceOf(WALLET1) - prevDaiWALLET1, amountWALLET1 * WAD);
```

Reply:
Done, but there were several places these types of things happened. Do a quick change to all of them.